### PR TITLE
hotfix: --enable-metrics params is always in invalid state

### PR DESCRIPTION
**Pull request by 1046102779**
# Description

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by beiwei30** [comment]
@1046102779 you should file an issue before submit a pull request, plus, for this particular proposed change, it looks like all E2E tests failed.

File: pkg/components/kubernetes_loader.go [hunk 1]
```diff
@@ -45,6 +45,7 @@ func NewKubernetesComponents(configuration config.KubernetesConfig, operatorClie
 func (k *KubernetesComponents) LoadComponents() ([]components_v1alpha1.Component, error) {
 	resp, err := k.client.ListComponents(context.Background(), &emptypb.Empty{}, grpc_retry.WithMax(operatorMaxRetries), grpc_retry.WithPerRetryTimeout(operatorCallTimeout))
 	if err != nil {
+		log.Warnf("daprd runtime error get components: %s", err)
```
Comment (yaron2) [inline pkg/components/kubernetes_loader.go:48]: Change to `error listing components: %s`

File: charts/dapr/charts/dapr_sentry/templates/dapr_sentry_deployment.yaml [hunk 2]
```diff
@@ -145,7 +145,7 @@ spec:
         - "--log-as-json"
 {{- end }}
 {{- if eq .Values.global.prometheus.enabled true }}
-        - "--enable-metrics"
+        - "--enable-metrics=true"
```
Comment (yaron2) [inline charts/dapr/charts/dapr_sentry/templates/dapr_sentry_deployment.yaml:148]: This should probably be
```
- "--enable-metrics"
- "true"
```

**Review by yaron2 — CHANGES_REQUESTED** [review]

File: charts/dapr/charts/dapr_sentry/templates/dapr_sentry_deployment.yaml (hunk 2 above)
Comment (daixiang0) [reply charts/dapr/charts/dapr_sentry/templates/dapr_sentry_deployment.yaml:148]: @1046102779 please update as comment said.

**Comment by 1046102779** [comment]
image

Dapr version v1.2.0 , when i change enable-metrics's value to false ,  it seems to be normal.

image

fixed in dapr v1.2.0.

**Pull request closed without merging**